bundles/framework/layerlist/view/LayerCollapse.jsx

```diff
@@ -37,10 +37,17 @@ export const LayerCollapse = ({ groups, openGroupTitles, filtered, selectedLayer
     return (
         <StyledCollapse bordered activeKey={openGroupTitles} onChange={keys => mutator.updateOpenGroupTitles(keys)}>
             {
-                panels.map(({group, showLayers}) => {
-                    const panelProps = {group, showLayers, selectedLayerIds, mapSrs, mutator, locale};
+                panels.map(({ group, showLayers }) => {
+                    const selectedLayersInGroup = selectedLayerIds.filter(cur => showLayers.map(lyr => lyr.getId()).includes(cur));
```

Comment:
Only pass ids that the component is interested in.

Reply:
Add this as a comment on the code?